vividus-plugin-web-app/src/test/java/org/vividus/ui/web/RuntimeEnvironmentTests.java

```diff
@@ -17,38 +17,55 @@
 package org.vividus.ui.web;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
 
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.InjectMocks;
 import org.mockito.Mock;
-import org.mockito.Mockito;
 import org.mockito.junit.jupiter.MockitoExtension;
 import org.vividus.ui.web.action.IJavascriptActions;
 
 @ExtendWith(MockitoExtension.class)
 class RuntimeEnvironmentTests
 {
-    private static final String USER_AGENT = "userAgent";
-
     @Mock
-    private IJavascriptActions jsActions;
+    private IJavascriptActions javascriptActions;
 
     @InjectMocks
     private RuntimeEnvironment runtimeConfiguration;
 
     @Test
     void testGetUserAgent()
     {
-        Mockito.when(jsActions.getUserAgent()).thenReturn(USER_AGENT);
-        assertEquals(USER_AGENT, runtimeConfiguration.getUserAgent());
+        String userAgent = "userAgent";
+        when(javascriptActions.getUserAgent()).thenReturn(userAgent);
+        assertEquals(userAgent, runtimeConfiguration.getUserAgent());
     }
 
     @Test
-    void testGetUserAgentMultipleCalls()
+    void testGetUserAgentMultipleTimes()
     {
         runtimeConfiguration.getUserAgent();
         runtimeConfiguration.getUserAgent();
-        Mockito.verify(jsActions).getUserAgent();
+        verify(javascriptActions, times(1)).getUserAgent();
```

Comment:
times(1) - no need?

Reply:
I'd like to keep it explicit to emphasize that exactly one invocation is expected, that's important in this test